packages/react-components/react-tree/src/hooks/useRovingTabIndexes.ts

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import { HTMLElementWalker } from './useHTMLElementWalker';
+
+interface RovingElement extends HTMLElement {
+  __fuiRoving: {
+    currentElement: HTMLElement;
+  };
+}
+
+/**
+ * https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/#kbd_roving_tabindex
+ */
+export function useRovingTabIndex(walker: HTMLElementWalker) {
+  const ref = React.useRef<RovingElement>(null);
+
+  function reset() {
```

Comment:
There should also be a way to set the default tabbable in a roving tabindex. Example: user switches from the Teams channel pane to chat pane and the default tabbable element should be the active chat

Reply:
Done, now if a tabIndex is set on the element, that element will maintain that tabindex as the default one